src/lib/services/server/kv/index.ts

```diff
@@ -107,6 +111,15 @@ export const createClient = async ({
     deleteUserInfoByToken: async (token) => {
       await redisClient.del(`${keys.userInfoByToken}:${token}`);
     },
+    setFireWeatherData: async (fireData) => {
+      const result = await redisClient.set(`${keys.fireWeatherData}`, JSON.stringify(fireData));
+      if (result !== "OK") throw new Error("could not set fire weather data in KV store");
+    },
+    getFireWeatherData: async () => {
+      const json = await redisClient.get(`${keys.fireWeatherData}`);
+      if (!json) return json;
```

Comment:
good point. as it stands, this has the potential to be a silent failure of sorts

Reply:
glad you mentioned this, because after some further digging, I discovered that _all_ of the KV methods are wrapped by [this function that is catching any errors](https://github.com/la-ldaf/ldaf-site/pull/564/files#diff-b316ad41586de17f1e6dea561b951373f6a67b3ec8045a16aed1d25044f0ebfbR70-R79). It's logging them, so we'll at least have access to them in that way, but I don't love the effects that has on downstream error handling.

It might have been a way to make development a little less disruptive as we were ironing out the kinks of getting the KV stuff working, but maybe something we want to revisit now